src/main/java/nl/ordina/jobcrawler/controller/SkillController.java

```diff
@@ -0,0 +1,123 @@
+package nl.ordina.jobcrawler.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import nl.ordina.jobcrawler.model.Skill;
+import nl.ordina.jobcrawler.model.SkillDTO;
+import nl.ordina.jobcrawler.repo.SkillRepository;
+import nl.ordina.jobcrawler.service.MatchSkillsService;
+import nl.ordina.jobcrawler.service.SkillService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+@Slf4j
+@CrossOrigin
+@RestController
+public class SkillController {
+
+
+    private final SkillService skillService;
+
+    private final MatchSkillsService matchSkillsService;
+
+    private final SkillRepository skillRepository;
```

Comment:
Controller layer should not communicate with the repository layer directly

Reply:
ok, will adapt